test/es6/moduletest2_mod0.js

```diff
@@ -0,0 +1,9 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+import './moduletest2_mod1a.js';
+import './moduletest2_mod1b.js';
+
+loaded++;
+if (loaded == 5) console.log("Pass");

```

Comment:
Should this log "Fail" if loaded != 5? Don't remember if empty output is treated as passing or not.